packages/dialtone-css/lib/build/less/components/tooltip.less

```diff
@@ -54,6 +54,7 @@
     line-height: var(--tooltip-line-height);
     letter-spacing: calc(var(--dt-space-100) * 0.25);
     text-align: center;
+    word-wrap: break-word;
```

Comment:
This doesn't affect tooltips with normal long text, but it does prevent the text being cut when there's a long text without spaces, so I think it's ok to add it to all tooltips.

Reply:
I think it's safe doing it for all, because it doesn't affect normal long text with spaces. I cuts the word only when the text reaches it's max width with a long word without spaces, instead of overflowing. And I think that's what we want in all cases

Reply:
I finally removed this because I added the spaces to the token value, so I no longer need it